backend/src/server/routes/est/certificate-est-router.ts

```diff
@@ -0,0 +1,160 @@
+import bcrypt from "bcrypt";
+import { z } from "zod";
+
+import { getConfig } from "@app/lib/config/env";
+import { BadRequestError, UnauthorizedError } from "@app/lib/errors";
+import { readLimit, writeLimit } from "@app/server/config/rateLimiter";
+
+export const registerCertificateEstRouter = async (server: FastifyZodProvider) => {
+  const appCfg = getConfig();
+
+  // add support for CSR bodies
+  server.addContentTypeParser("application/pkcs10", { parseAs: "string" }, (_, body, done) => {
+    try {
+      done(null, (body as string).replace(/\n/g, "").replace(/ /g, ""));
```

Comment:
Why do we need `.replace(/\n/g, "").replace(/ /g, ""))` here? — This caused the PEM-encoded CSR to be malformed.

Removing it worked for me.

Reply:
BouncyCastle EST client sends the CSR with "/n" and spaces like the ff:
<img width="902" alt="image" src="https://github.com/user-attachments/assets/79c73ad5-f5e1-49fb-8832-a7e1bf628e25">

If we remove this, we get the following error during CSR parsing
<img width="880" alt="image" src="https://github.com/user-attachments/assets/9a4bd496-2d6c-4847-8c5d-3d2f97617626">


How are you sending the CSR in your tests?

Reply:
As per the sample usage in the RFC, it's sending over the CSR without the begin and end tags
https://datatracker.ietf.org/doc/html/rfc7030#appendix-A.3

<img width="701" alt="image" src="https://github.com/user-attachments/assets/8511e84b-39da-4314-b157-e128e93b2423">